lib/lang/latin/latin-language-dataset.js

```diff
@@ -273,17 +273,29 @@ export default class LatinLanguageDataset extends LanguageDataset {
   // for Lemmas
   addVerbForms (partOfSpeech, data) {
     // First row are headers
+    this.verbsIrregularLemmas = []
+
     for (let i = 1; i < data.length; i++) {
       const item = data[i]
       let lemma = item[0]
       // let principalParts = item[1].split(/_/)
+      let principalParts = item[1]
+
       let form = item[2]
 
       // Lemma,PrincipalParts,Form,Voice,Mood,Tense,Number,Person,Footnote
       let features = [
         partOfSpeech,
         this.features.get(Feature.types.fullForm).createFromImporter(lemma)
       ]
+
+      if (principalParts) {
+        features.push(this.features.get(Feature.types.hdwd).createFromImporter(principalParts))
```

Comment:
See note above, the feature value would, I think, be either the Lemma object itself, or a list of the hdwd and the list of prinicipal parts. (e.g. "eo, ire, ivi(ii), itus")